bugbot/rules/lot_of_comments.py

```diff
@@ -0,0 +1,53 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from bugbot import utils
+from bugbot.bzcleaner import BzCleaner
+
+
+class SeveralComments(BzCleaner):
+    def __init__(self):
+        super(SeveralComments, self).__init__()
```

Comment:
nit: We do not support Python 2 anymore; consider using Python 3 style super() without arguments:
```suggestion
        super().__init__()
```

We still have many classes with the old style. We should refactor that at some point for consistency.